### PR TITLE
Pre-release v4.3.0-alpha.1

### PR DESCRIPTION
**Pull request by miaulalala**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
 # Changelog
+## 4.3.0 - 2023-02-16
+### Added
+- File attachments for calendar events
+### Changed
+- Lazy load dashboard component
+- Wider input for recurrences
+### Fixed
+- Exception handing for booking controller
+- can_subscribe_link fallback
```
Comment (GretaD) [inline CHANGELOG.md:11]: i would remove this as looks a bit technical

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,22 @@
 
 # Changelog
+## 4.3.0 - 2023-02-16
```
Comment (ChristophWurst) [inline CHANGELOG.md:3]:
```suggestion
## 4.3.0 - unreleased
```

File: CHANGELOG.md (hunk 1 above)
Comment (ChristophWurst) [reply CHANGELOG.md:11]: Marked as resolved but not resolved

Comment (miaulalala) [reply CHANGELOG.md:11]: https://github.com/nextcloud/calendar/pull/4988/commits/c1d87cf029515a028ff9358ae5949eb55cb7498b

Comment (ChristophWurst) [reply CHANGELOG.md]: check again

Comment (miaulalala) [reply CHANGELOG.md]: wrong line, fixed now

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**